.gitignore

```diff
@@ -1,3 +1,46 @@
 *.o
 *~
 txtempus
+
+# Created by https://www.toptal.com/developers/gitignore/api/linux,visualstudiocode
+# Edit at https://www.toptal.com/developers/gitignore?templates=linux,visualstudiocode
+
+### Linux ###
+*~
+
+# temporary files which can be created if a process still has a handle open of a deleted file
+.fuse_hidden*
+
+# KDE directory preferences
+.directory
+
+# Linux trash folder which might appear on any partition or disk
+.Trash-*
+
+# .nfs files are created when an open file is removed but is still being accessed
+.nfs*
+
+### VisualStudioCode ###
+.vscode/*
+# !.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+!.vscode/*.code-snippets
+
+# Local History for Visual Studio Code
+.history/
+
+# Built Visual Studio Code Extensions
+*.vsix
+
+### VisualStudioCode Patch ###
+# Ignore all local history of files
+.history
+.ionide
+
+# Support for Project snippet scope
+
+# End of https://www.toptal.com/developers/gitignore/api/linux,visualstudiocode
+
+build/*
```

Comment:
This does look like a lot of IDE gnerated stuff in here that should not necessarily make it upstream; every developer with specific IDEs should keep their .gitignore with whatever is needed in theri context.

So here, we can just simplify the additions to `.gitignore` to`build/*` to accomodate the cmake-project.